app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -291,85 +291,76 @@ def test_experiment_bug_successfully_created(self):
             ),
         )
 
-    def test_bugzilla_error_doesnt_create_notification(self):
+    def test_bugzilla_error_doesnt_create_notifications(self):
```

Comment:
Ah yeah, I think we want the opposite behaviour of this now:

https://github.com/mozilla/experimenter/issues/1282#issuecomment-494567539

Reply:
I would say leave this as is in here (ie consistent with the rest of the code) and do those changes in separately in #1282.  Don't wanna cram too much into one PR.

Reply:
will leave this be for now, there's probably too much in this PR already